test/socket/chat.js

```diff
@@ -0,0 +1,82 @@
+const chai = require('chai')
+const should = chai.should()
+const expect = chai.expect
+const chaiHttp = require('chai-http')
+chai.use(chaiHttp)
+
+const helpers = require('./../support/helpers')
+const { User, Chat, ChatMessage } = require('../../models/index')
+const app = require('../../server')
+
+const io = require('socket.io-client');
+
+const socketURL = `http://${process.env.CHAT_HOST}:${process.env.CHAT_PORT}`;
+
+describe('chats socket ', function () {
+
+    let clientConnected = `client:connected`;
+    let clientSendMessage = `client:sendMessage`;
+    let serverConnected = `server:connected`;
+    let serverGotMessage = `server:gotMessage`;
+
+    var superUser = {name:"ella", id:1};
+    var chatClient1 = {name:"scrappy", id:2};
+
+    it('clientId as userId is returned correctly from server', function (done){
```

Comment:
Consider some more appropriate BDD 'storytelling': 
`it('should correctly return clientId as userId from server', ...)`

Same goes for the rest of the tests